pyomo/core/tests/transform/test_add_slacks.py

```diff
@@ -357,7 +357,7 @@ def test_transformed_constraints_sumexpression_body(self):
 
     def test_transformed_constraint_scalar_body(self):
         m = self.makeModel()
-        m.p = Param(initialize=6)
+        m.p = Param(initialize=6, mutable=True)
```

Comment:
Because the expression given to the constraint resolves to a `bool` if the `Param` is not mutable. It did not seem like it was important for this test to use an immutable `Param`, but maybe that was an incorrect conclusion.